aten/src/ATen/cpu/vec256/vec256_int.h

```diff
@@ -12,6 +12,11 @@ namespace {
 struct Vec256i {
 protected:
   __m256i values;
+
+  static inline __m256i invert(__m256i v) {
```

Comment:
I think this may need to be `__m256i invert(const __m256i& v) {` or it may generate some warnings on certain versions of GCC.